docs/content/_index.md

```diff
@@ -10,16 +10,33 @@ draft: false
 
 ## What is Guardicore Infection Monkey?
 
-The Infection Monkey is an open source security tool for testing a data center's resiliency to perimeter breaches and internal server infection. The Monkey uses various methods to self propagate across a data center and reports success to a centralized Monkey Island Command and Control server.
+The Infection Monkey is an open source breach and attack simulation tool for testing a data center's resiliency to perimeter breaches and internal server infection.
+Infection Monkey will help you test implemented security solutions and will provide visibility of the internal network through the eyes of an attacker.
```

Comment:
```suggestion
Infection Monkey will help you validate your existing security solutions and will provide a view of your network from an attacker's perspective.
```

Reply:
Fixed, but left the word _internal_, because for monkey to work effectively it does need to be  in an internal network.